### PR TITLE
feat(xy): bar chart supports mouse scrolling

### PR DESCRIPTION
**Pull request by eraop**
## Description

> Added mouse scrolling configuration and default display size configuration for bar chart.

## Screenshots

- [x] This PR does not have any UI modifications  

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation or example update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation and example

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@momentum-design/widgets",
-  "version": "1.0.0-alpha.15",
```
Comment (bndynet) [inline package.json:3]: Please do not change the version number manually.

File: package.json [hunk 2]
```diff
@@ -122,8 +122,8 @@
     "chart.js": "^4.3.0",
     "chartjs-adapter-moment": "^1.0.1",
     "chartjs-chart-wordcloud": "^4.3.2",
+    "chartjs-plugin-zoom": "^2.0.1",
     "chroma-js": "^2.4.2",
-    "html2canvas": "^1.4.1",
```
Comment (bndynet) [inline package.json:126]: Please confirm whether you removed this package by using `npm uninstall` command.

File: src/lib/xy/xy.ts [hunk 3]
```diff
@@ -73,14 +76,18 @@ export abstract class XYChart extends Chart<XYData, XYChartOptions> {
     if (this.chartData) {
       chartDatasets = this.getDatasets();
     }
+    let plugins: CJPlugin[] = [chartA11y];
+    if (this.options.scrollable) {
+      plugins = [chartA11y, zoomPlugin];
```
Comment (bndynet) [inline src/lib/xy/xy.ts:81]: Here you just push into the **plugins** for zoomPlugin.

File: website/docs/tutorials/charts/bar.mdx [hunk 4]
```diff
@@ -55,10 +55,12 @@ function showComponent() {
     },
   ];
   const options = {
+    scrollable: true,
     categoryAxis: {
       gridDisplay: true,
       enableColor: true,
       dataKey: 'country',
+      defaultSize: 6,
```
Comment (bndynet) [inline website/docs/tutorials/charts/bar.mdx:63]: What is the unit for **defaultSize**? **px**?

File: package.json (hunk 2 above)
Comment (eraop) [reply package.json:126]: yes

File: src/lib/xy/xy.ts (hunk 3 above)
Comment (eraop) [reply src/lib/xy/xy.ts]: fixed

File: website/docs/tutorials/charts/bar.mdx (hunk 4 above)
Comment (eraop) [reply website/docs/tutorials/charts/bar.mdx]: I renamed it to **maxLabels**.

**Review by bndynet — APPROVED** [review]

**Pull request merged**